cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/DistributedPreviewManager.java

```diff
@@ -172,6 +173,14 @@ public void run() {
                                     new StatefulDisk("preview-runner-data", diskSize, "/data"));
           }
 
+          if (twillPreparer instanceof SecureTwillPreparer) {
+            String twillUserIdentity = cConf.get(Constants.Security.TwillIdentity.IDENTITY_USER);
```

Comment:
Is it user identity or identity user?

Reply:
It's intended to be IDENTITY_USER; I want to preserve prefix since it'll preserve alphabetical ordering.

Reply:
Maybe just call it TwillIdentity.USER, the prefix seems redundant

Reply:
I am referring to the variable name as `twillUserIdentity`, while the config is `IDENTITY_USER`. It is better to be consistent with one way or the other.